test/bin.spec.js

```diff
@@ -234,7 +234,7 @@ test('w3 remove - no such upload', async t => {
 
   const service = mockService({
     upload: {
-      remove: provide(UploadCapabilities.remove, () => {})
+      remove: provide(UploadCapabilities.remove, () => { })
```

Comment:
You monster

Reply:
LOL. I didn't mean it I swear! VSCode too aggressive!

Reply:
undid this!